shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/bytebuddy/transformer/ShardingSphereTransformer.java

```diff
@@ -145,28 +132,32 @@
         return result;
     }
     
+    private ShardingSphereTransformationPoint<?> getStaticClassTransformationPoint(final MethodDescription.InDefinedShape md, final List<ClassStaticMethodPoint> classStaticMethodAroundPoints) {
+        List<ClassStaticMethodPoint> classStaticMethodPoints = classStaticMethodAroundPoints.stream().filter(point -> point.getMatcher().matches(md)).collect(Collectors.toList());
+        if (classStaticMethodPoints.isEmpty()) {
+            return null;
+        }
+        if (classStaticMethodPoints.size() == 1) {
+            return new ShardingSphereTransformationPoint<>(md, new ClassStaticMethodAroundInterceptor(pluginLoader.getOrCreateInstance(classStaticMethodPoints.get(0).getAdvice())));
+        } else {
+            Collection<ClassStaticMethodAroundAdvice> classStaticMethodAroundAdvices = new LinkedList<>();
+            for (ClassStaticMethodPoint point : classStaticMethodPoints) {
+                if (null != point.getAdvice()) {
+                    classStaticMethodAroundAdvices.add(pluginLoader.getOrCreateInstance(point.getAdvice()));
+                }
+            }
+            return new ShardingSphereTransformationPoint<>(md, new ComposeClassStaticMethodAroundInterceptor(classStaticMethodAroundAdvices));
```

Comment:
Please rename `md` to `methodDescription` too.
It is better to do the global search

Reply:
thanks